docs/src/main/asciidoc/datasource-guide.adoc

```diff
@@ -105,6 +105,52 @@ include::../../../../extensions/agroal/deployment/src/test/resources/application
 
 There are other configuration options, detailed below.
 
+=== JDBC URL configurations
+
+As noted above, {project-name} currently supports four databases.
+Each of those databases contains different JDBC URL configuration options.
+Going into each of those options is beyond the scope of this document, but it give an overview of each database URL and link to the official documentation.
+
+==== H2
+
+`jdbc:h2:{ {.|mem:}[name] | [file:]fileName | {tcp|ssl}:[//]server[:port][,server2[:port]]/name }[;key=value...]`
+
+H2 is an embedded database.
+It can run as a server, based on a file, or live completely in memory.
+All of these options are available as listed above.
+You can find more information, visit the https://h2database.com/html/features.html?highlight=url&search=url#database_url[official documentation].
```

Comment:
This sentence seems weird to me (but I'm not a native speaker). Maybe `To find more information, visit...`?